File: Baekjoon/solved_ac/class2/10989.cpp
// 10989: 수 정렬하기 3

#include <iostream>
using namespace std;

int main() {
  int N, cnt[10000] = {};
  cin >> N;

  while (N--) {
    int num;
    
    cin >> num;
    cnt[num-1]++;
  }

  for (int i=0; i<10000; i++) {
    if (cnt[i] != 0) {
      for (int j=0; j<cnt[i]; j++) cout << i+1 << '\n';
    }
  }
}

File: Baekjoon/solved_ac/class2/10989.py
# 10989: 수 정렬하기 3

N = int(input())

cnt = [0 for _ in range(10000)]

for _ in range(N):
  cnt[int(input())-1] += 1

for i in range(10000):
  if (cnt[i] != 0):
    for j in range(cnt[i]):
      print(i+1)
